.github/workflows/kind-e2e.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
         # Build and Publish our containers to the docker daemon (including test assets)
         export GO111MODULE=on
         export GOFLAGS=-mod=vendor
-        ko apply --platform=all -P \
+        ko apply -P \
```

Comment:
Just curious about this removal?

Reply:
> Drop --platform=all during actions, this is now 4 architectures, which can get really slow.

From the PR description.  distroless now ships amd64, arm64, s390x, and powerpc 😎

Reply:
Sorry, I know we've been getting more requests to run tests for different platforms. So, is the plan to tackle this later, or ?

Reply:
Here it will always be overkill (we're building for 4 platforms while testing for one), I just wanted some mileage through the build process early on, which is why I'd sprinkled this a few places (we're pulling those out many places now).

As for longer-term story, we need the capacity to spin up clusters for these architectures, which sadly also means we need to solve the non-GCP infrastructure problem.  There are a few slack threads discussing this now, I'll tag you in there.

Reply:
Ack, makes sense 👍